scopes/react/react-native/webpack/webpack-transformers.ts

```diff
@@ -1,17 +1,31 @@
 import { WebpackConfigTransformer, WebpackConfigMutator, WebpackConfigTransformContext } from '@teambit/webpack';
 
+const reactNativeElementRule = {
```

Comment:
maybe remove the "Element" from the name, because there is a library called react-native-elements and if the rule is not especially for this library we can misunderstand the rule.

Reply:
Done